testing.go

```diff
@@ -193,8 +216,8 @@ func testClient() *Client {
 		)
 }
 
-func initTest() (server *Server, client *Client, start, stop func()) {
-	deleteQueue(defaultTestQueue) // Ensure queue is clean from the start.
+func initTest(t *testing.T) (server *Server, client *Client, start, stop func()) {
+	deleteAllQueues(t)
```

Comment:
Huh, surprised we did this, does this mean we can never run the tests in parallel?

Reply:
Yes, this makes the tests less flaky. But we can probably support parallel tests if we ensure to create queues with random names.